### PR TITLE
Fix wrong example for LabelsPresence

### PR DESCRIPTION
**Pull request by nak3**

### CODE REVIEW

File: admin_guide/scheduler.adoc [hunk 1]
```diff
@@ -61,7 +61,7 @@ These predicates can be configured by the user to tweak their functioning.  They
 ----
 **_LabelsPresence_** checks whether a particular node has a certain label defined or not, regardless of its value.
 ----
-{"name" : "ZoneRequired", "argument" : {"serviceAffinity" : {"labels" : ["zone"]}}}
+{"name" : "ZoneRequired", "argument" : {"labels" : ["retiring"], "presence" : true}}}
```
Comment (adellape) [inline admin_guide/scheduler.adoc]: I think there's an extra `}` on the end here?

**Comment by adellape** [comment]
Other than possibly the one inline comment, this fix LGTM.

@abhgupta Can you ack?

File: admin_guide/scheduler.adoc (hunk 1 above)
Comment (abhgupta) [inline admin_guide/scheduler.adoc]: Yes, there is an extra `}` in there after the change.

However, to make the example more meaningful, I would suggest that you change the presence from "true" to "false". With presence set to "true", it currently reads as: "Consider ONLY those nodes that have the `retiring` label present." Instead, we want the opposite and hence the "false" value is required.

**Comment by abhgupta** [comment]
Other than the comment that I made, LGTM. Thanks for fixing this.

**Comment by nak3** [comment]
Thank you. I fixed it.

**Pull request merged**